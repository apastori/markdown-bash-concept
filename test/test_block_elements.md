# Heading 1

This is a paragraph.

## Heading 2

> This is a blockquote.
>
> > This is a nested blockquote.

### Heading 3

- Unordered list item 1
- Unordered list item 2
  - Nested unordered list item
- Unordered list item 3

#### Heading 4

1. Ordered list item 1
2. Ordered list item 2
   1. Nested ordered list item
3. Ordered list item 3

##### Heading 5

```bash
# This is a bash code block
echo "Hello, World!"
```

    # This is an indented code block
    print("Hello, World!")

###### Heading 6

---

This is another paragraph.

And another one.
