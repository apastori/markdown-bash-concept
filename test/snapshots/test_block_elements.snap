\"# Tokenizer Snapshot\"  
Tokenizing file: test/test_block_elements.md
HEADING|1:Heading 1|1| EMPTY_LINE||2| PARAGRAPH|This is a paragraph.|3| EMPTY_LINE||4| HEADING|2:Heading 2|5| EMPTY_LINE||6| BLOCKQUOTE|This is a blockquote.|7| BLOCKQUOTE||8| BLOCKQUOTE|> This is a nested blockquote.|9| EMPTY_LINE||10| HEADING|3:Heading 3|11| EMPTY_LINE||12| UNORDERED_LIST_ITEM|Unordered list item 1|13| UNORDERED_LIST_ITEM|Unordered list item 2|14| UNORDERED_LIST_ITEM|Nested unordered list item|15| UNORDERED_LIST_ITEM|Unordered list item 3|16| EMPTY_LINE||17| HEADING|4:Heading 4|18| EMPTY_LINE||19| ORDERED_LIST_ITEM|Ordered list item 1|20| ORDERED_LIST_ITEM|Ordered list item 2|21| ORDERED_LIST_ITEM|Nested ordered list item|22| ORDERED_LIST_ITEM|Ordered list item 3|23| EMPTY_LINE||24| HEADING|5:Heading 5|25| EMPTY_LINE||26| CODE_BLOCK|start:bash:```|27| HEADING|1:This is a bash code block|28| PARAGRAPH|echo "Hello, World!"|29| CODE_BLOCK|start::```|30| EMPTY_LINE||31| CODE_BLOCK|content:# This is an indented code block|32| CODE_BLOCK|content:print("Hello, World!")|33| EMPTY_LINE||34| HEADING|6:Heading 6|35| EMPTY_LINE||36| HORIZONTAL_RULE||37| EMPTY_LINE||38| PARAGRAPH|This is another paragraph.|39| EMPTY_LINE||40| PARAGRAPH|And another one.|41|
